app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
```

Comment:
We don't want to do .order in the controller as it is a more complex data operation besides simple CRUD. we would have liked to see it in a model method (see comment in Movie model)